### ISSUE TITLE
Shift+Tab Does Not Show Answer on Fill-in-the-Blanks Cards

### ISSUE THREAD

**Issue by deron-dev** [Yvee1/hascard#35]
A sample from the deck I am using:

```markdown
<!-- ... -->

---

# se
_oneself_

---

# sé
_I know_

---

<!-- ... -->
```

When I press Shift+Tab, the answer is not shown.

Running hascard version `0.6.0.1` on Linux.

Tried on `alacritty`, `xfce4-terminal`, and `kitty`.

**Comment by Yvee1** [comment]
Thanks for letting me know. I tested it on my current setup and I have the same issue, though it used to work. I don't think there is a good key combination that will work reliably. My plan now is to just show the correct answer when one enters nothing or when the same answer is submitted twice in a row. I'll try to implement this soonish and push a fix.

**Comment by Yvee1** [comment]
I implemented the fix described in my previous comment and released a new version, v0.6.0.2. Hope this helps!

**Comment by deron-dev** [comment]
Just tried it. Helps it does! Thanks :)
